src/logic/useScooters.ts

```diff
@@ -53,9 +54,24 @@ export default function useScooters(): Record<
         const intervalId = setInterval(() => {
             fetchScooters(settings).then(setScooters)
         }, REFRESH_INTERVAL)
+        return (): void => clearInterval(intervalId)
+    }, [scooters, settings])
+
+    return scooters
+}
 
+export function useOperators(): Record<ScooterOperator, Scooter[]> | null {
```

Comment:
Er det nødvendig med eigen hook for dette? Einaste forskjellen eg ser er at `countScootersByOperator` blir brukt. Då tenker eg det heller er bedre å bruke `useScooters` + `countScootersByOperator` der det er behov.

Reply:
Fixed